### ISSUE TITLE
Error shrink_it inkscape

### ISSUE THREAD

**Issue by Storm21CH** [bzhxx/LCD-Game-Shrinker#20]
Hello have this error... what is wrong? Installed inkscape two times. Thank you!

`storm@StormMobile-PC:~/LCD-Game-Shrinker$ python3 shrink_it.py
Parachute                |▆▆▆▆▆▆▆▆▆▆▆▆▆-------| 2/3 Shrink segmentsTraceback (most recent call last):
  File "shrink_it.py", line 497, in <module>
    inkscape_output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL,shell=True)
  File "/usr/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'inkscape ./build/gnw_pchute/segments.svg --export-type=png --export-overwrite --export-background=#FFFFFF --export-background-opacity=1 --export-type=png -o ./build/gnw_pchute/gnw_pchute.png' returned non-zero exit status 134.
storm@StormMobile-PC:~/LCD-Game-Shrinker$`

**Comment by bzhxx** [comment]
Hello
Have you installed Inkscape version 1.10 ?
is inkscape in PATH ?

**Comment by Storm21CH** [comment]
Thanks for your answer! I tried this guide...
[https://gist.github.com/DNA64/16fed499d6bd4664b78b4c0a9638e4ef](url)

`sudo add-apt-repository ppa:inkscape.dev/stable-1.1`
`sudo apt-get update`
`sudo apt install inkscape`

Should be 1.1 but when I type inkscape --version I get an error. I don't know if it should show the version like this.

`storm@StormMobile-PC:~$ inkscape --version
Unable to init server: Could not connect: Connection refused
Wrong __data_start/_end pair
Aborted (core dumped)
storm@StormMobile-PC:~$`

is inkscape in PATH ?

Don't quite understand, do I have to have inkscape in the LCD-Game-Shrinker folder? Or do you mean set an environment variable? Sorry Linux is not my friend...

**Comment by bzhxx** [comment]
It seems that inkscape is not correctly installed.
inkscape should be installed in /usr/bin/inkscape
echo $PATH to check if inkscape path is set.
inkscape --version should work.

**Comment by Storm21CH** [comment]
Hello
Yes! But...
I got it working after hours of trying. Failed with WSL Ubuntu 20.04 error from above and I can't get inkscape to run there. Failure with Win10 python requirements. Failed with VM and 20.04, other error messages. Then finally with WSL Ubuntu 22.04 everything went smoothly. Besides inkscape I just installed with "sudo apt install inkscape" and that's how it worked. Thank you for your efforts!

Greetings
